setup/rhel72-linuxonecc/ansible-playbook.yaml

```diff
@@ -0,0 +1,119 @@
+---
+- hosts: iojs-build-rhel72-linuxonecc
+
+  remote_user: root
+
+  tasks:
+
+    - include_vars: ansible-vars.yaml
+      tags: vars
+
+    - name: General | yum Update
+      yum: name=* state=latest
+      tags: general
+
+    - name: General | Install required packages
+      yum: name={{ item }} state=latest
+      with_items: packages
+      tags: general
+
+    - name: NTP | Run initial NTP
+      command: ntpdate -u pool.ntp.org
+      tags: ntp
+
+    - name: NTP | Enable NTP
+      command: chkconfig ntpd on
+      tags: ntp
+
+    - name: NTP | Start NTP
+      service: name=ntpd state=started enabled=yes
+      tags: ntp
+
+    - name: General | Increase file descriptor limits
+      lineinfile: dest=/etc/security/limits.conf line="{{ item }}"
+      with_items:
+        - "*         hard    nofile      500000"
+        - "*         soft    nofile      500000"
+        - "root      hard    nofile      500000"
+        - "root      soft    nofile      500000"
+      tags: general
+
+    - name: User | Add {{ server_user }} user
+      user: name="{{ server_user }}" shell=/bin/bash
+      tags: user
+
+    - name: User | check if larger disk has file system 
+      command: file -s /dev/dasdb1
```

Comment:
i guess this mountpoint could benefit from being abstracted into ansible-vars.yaml?